generator/internal/stub_generator.cc

```diff
@@ -111,11 +129,68 @@ Status StubGenerator::GenerateHeader() {
   // clang-format on
 
   // close abstract interface Stub base class
-  header_.Print(vars_,  // clang-format off
-    "};  // $stub_class_name$\n"
+  header_.Print(service_vars_,  // clang-format off
+    "};\n\n");
+  // clang-format on
+
+  // default stub class
+  header_.Print(
+      service_vars_,  // clang-format off
+    "class Default$stub_class_name$ : public $stub_class_name$ {\n"
+    " public:\n"
+    "  Default$stub_class_name$(\n"
+    "      std::unique_ptr<$grpc_stub_fqn$::StubInterface> grpc_stub,\n"
+    "      std::unique_ptr<google::longrunning::Operations::StubInterface> "
+    "operations)\n"
+    "      : grpc_stub_(std::move(grpc_stub)),\n"
+    "        operations_(std::move(operations)) {}\n\n");
+                      // clang-format on
+
+  for (int i = 0; i < service_descriptor_->method_count(); i++) {
```

Comment:
Same comments as above:

nit: `++i`
Question about safety of `op[]`.

And again below.